core/src/validator.rs

```diff
@@ -174,13 +173,12 @@ impl BlockVerificationMethod {
     }
 
     pub fn cli_message() -> &'static str {
-        lazy_static! {
-            static ref MESSAGE: String = format!(
+        static MESSAGE: std::sync::LazyLock<String> = std::sync::LazyLock::new(|| {
+            format!(
```

Comment:
I removed the lazy behavior for all of these in https://github.com/anza-xyz/agave/pull/6123; please rebase which should remove the need to touch this file altogether